content/technotes/2020-02-26-tic-ghactions.Rmd

```diff
@@ -19,44 +19,47 @@ tags:
   - GitHub
   - GitHub Actions
   - Continuous Integration
+  - software-peer-review
+  - tic
+  - community
 # The summary below will be used by e.g. Twitter cards
-description: "tic aims to simplify CI related tasks for R projects (not just packages by the way) and now provides supercharged support for GitHub Actions. By enhancing the good work of [r-lib/actions](https://github.com/r-lib/actions) tic provides a DSL for R for CI tasks."
+description: "tic aims to simplify Continuous Integration (CI) related tasks for R projects (not just packages by the way) and now provides supercharged support for GitHub Actions. By enhancing the good work of [r-lib/actions](https://github.com/r-lib/actions) tic provides a DSL for R for CI tasks."
 ---
 
-"Continuous Integration" (CI) has become a standard for proper software development.
+Continuous Integration (CI) has become a standard for proper software development.
```

Comment:
Could you define Continuous Integration here? Not just what the acroynym means, but what the term means as well. That'll help beginner R programmers who may be less familiar with this field.

Reply:
I've added a link to wikipedia. Is that sufficient? I am personally always skeptical whether such terms should be explained again from the ground up in a blog post (I know I might face different opinions here).

Reply:
I'm happy with this compromise :slightly_smiling_face: